_posts/2020-03-31-DevOpsDays-2019.md

```diff
@@ -0,0 +1,235 @@
+
+The market has changed over the recent decade. 
+Public and hybrid cloud are emerging and on-premise deployment is declining. 
+We see that especially AWS and Azure have been expanding in the recent years as more and more customers are using their services. 
+All those cloud providers like AWS, Google Cloud, Azure,... have their own APIs and those are not interchangeable since they are lacking a standard. 
+The fact that an API is open does not imply that it is a standard, it's only something you have access to. 
+That is why tools like [Terraform](https://www.terraform.io/){:target="_blank" rel="noopener noreferrer"} are becoming popular. 
+People have a need for a unified standard to manage their resources. 
+Bernd thinks that Open Source is very important to have open standards although those are not the same. 
+
+There is also a problem if a certain cloud provider uses Open Software. 
+If they use a certain piece of open source software, there is no direct connection between the customer and the original developer of that software, because that layer was cut. 
+No matter what business model, the creator of the software has to earn money.
+
+Open Source tools in the cloud make sure that you have direct contact to the creators of the software to help you out. 
+AWS, on the other hand, has their own versions of Open Source Software like Open Distro which is basically their own distribution of ElasticSearch. 
+This removes the connection with the original developer, because you can only contact them for support. 
+It brings fear and protection to the developers of the software because they are afraid that money and intellectual property goes away. 
+That’s the reason why they came up with own licenses like Server-Side Public License (SSPL) that MongoDB uses, for instance. 
+This tells you that if you alter it to sell it as a SaaS offering, you have to contribute to that code.
+And Redis prohibits you to sell their enterprise model as their cloud solution.
+
+Bernd doesn't like that people come up with new licenses, but he also understands that they are afraid that their money stream goes away, and they see this as a solution. 
+We have to see that we support enough Open Source and Open Standards, because they need it to improve and maintain their software or service.
+
+In today’s oligopoly of cloud providers, it’s really difficult to new players to emerge their new business, although this will be good to have a more diverse internet. 
+Most companies are focusing on the more human side of things when it comes to diversity. 
+He compares it to when there are only a few companies that control the availability of Insulin because there’s no market law to reduce prices. 
+It would be a lot cheaper if there were more players that provide Insulin.
+“Abuse of Power comes as no surprise”.
+
+At the end of his talk, he draws some conclusions:
+
+**Be reasonable**
+  * Think about what you are doing
```

Comment:
Nitpicking, but I'd add a period at the end just to be in line with the other bullets. :)

Reply:
Fixed :)